src/TLuaInterpreter.cpp

```diff
@@ -5724,6 +5798,197 @@ int TLuaInterpreter::loadSoundFile(lua_State* L)
     return loadMediaFileAsOrderedArguments(L);
 }
 
+// Private
+int TLuaInterpreter::playVideoFileAsOrderedArguments(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    TMediaData mediaData{};
+    int numArgs = lua_gettop(L);
```

Comment:
We tend to use `n` for this value in other Lua functions in this file (and, thinking about it, `s` for what you called `counter` further up)...
```suggestion
    int n = lua_gettop(L);
```